plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -388,11 +370,25 @@ def __init__(self, *args, **kwargs):
         self.raw_xml = ''
 
     @property
+    def relative_path(self):  # TODO: what is this used for?
```

Comment:
The `unit.relative_path` and `unit.download_path` is a little *helter-skelter* in pulp_rpm.  They are referenced in the downloading code.  I suspect this impl (which I grabbed from the pre-mongoengine model) may be accounting for downloading the same rpm file name (but different checksum) for two different units into the working directory concurrently.

I think given our improved method for storing in the platform, it's worth getting rid of both properties.  The code doing the downloading can download the file to a random file name and it will still get stored in the platform correctly.  The `FileContentUnit.import_content()` will do this properly since the `FileContentUnit._storage_path` is based on the unit ID and will include the file name.